infrastructure/smart-contracts/contracts/upgrades/node-license/NodeLicense8.sol

```diff
@@ -548,6 +596,11 @@ contract NodeLicense8 is ERC721EnumerableUpgradeable, AccessControlUpgradeable
             IERC20 token = IERC20(esXaiAddress);
             token.transfer(msg.sender, rewardEsXai);
         }
+        if(rewardUSDC > 0){
+            IERC20 token = IERC20(usdcAddress);
+            token.transfer(msg.sender, rewardUSDC);
+            emit RewardClaimedUSDC(msg.sender, rewardUSDC);
+        }
         emit RewardClaimed(msg.sender, reward, rewardXai, rewardEsXai); 
```

Comment:
Should these be merged into a single event? (add usdc param to existing event)?

Otherwise `emit RewardClaimed(msg.sender, reward, rewardXai, rewardEsXai); ` may be triggered with all 0 values?

Reply:
I considered that but went with a separate event to avoid any breaking changes to services relying on the event signature. But if that's not a concern I can update the event with an extra param. Thoughts?

Reply:
I'll defer to @CryptITAustria but I think the added `rewardXai, rewardEsXai` params are already a breaking change and have not been deployed yet so this would be the perfect time to add yours if that is the case.

Reply:
Sounds like the right call, I went ahead and made the changes. If we decide the other way is better I can revert it 👍

Reply:
@craigbranscom, @XDapps is correct, we already extended the RewardClaimed with TK updates for introducing mint with xai/esXai, so you can extend the existing. I hope we are done adding payment tokens...

The claimed event should not be of much impact, its not used within our infrastructure, there is a chance it might be used with the dune queries but these would have to be updated anyway to reflect the other payment tokens.

Reply:
I think we should just emit the reward token address and the amount. It would be address(0) for AETH. Let me know what you think ?